packages/siop-oid4vp/lib/authorization-response/Dcql.ts

```diff
@@ -71,39 +92,39 @@ export class Dcql {
     return DcqlPresentationResult.fromDcqlPresentation(dcqlPresentation, { dcqlQuery })
   }
 
-  static toDcqlMdocCredential = (vc: WrappedMdocCredential): DcqlMdocCredential => {
+  static toDcqlMdocCredential = (vc: MdocDocument): DcqlMdocCredential => {
     return {
       credential_format: 'mso_mdoc',
-      doctype: vc.credential.toJson().docType,
-      namespaces: vc.decoded,
+      doctype: vc.toJson().docType,
+      namespaces: getMdocDecodedPayload(vc),
       cryptographic_holder_binding: hasCryptographicHolderBinding('mso_mdoc', vc),
     } satisfies DcqlMdocCredential
   }
 
-  static toDcqlSdJwtCredential = (vc: WrappedSdJwtVerifiableCredential): DcqlSdJwtVcCredential => {
+  static toDcqlSdJwtCredential = (vc: SdJwtDecodedVerifiableCredential): DcqlSdJwtVcCredential => {
     return {
       credential_format: 'dc+sd-jwt',
-      vct: vc.decoded.vct,
-      claims: vc.decoded,
+      vct: vc.decodedPayload.vct,
+      claims: vc.decodedPayload,
       cryptographic_holder_binding: hasCryptographicHolderBinding('dc+sd-jwt', vc),
     } satisfies DcqlSdJwtVcCredential
   }
 
-  static toDcqlJwtCredential = (vc: WrappedW3CVerifiableCredential): DcqlW3cVcCredential => {
+  static toDcqlJwtCredential = (vc: JwtDecodedVerifiableCredential): DcqlW3cVcCredential => {
```

Comment:
The previous one accepted any W3C VCDM 1.1 credential. The current one only accepts Jwt Decoded. Is there a specific reason for it?

Reply:
reverted this, using wrappped again